Makefile.custom.mk

```diff
@@ -6,16 +6,11 @@ clean-dry-run: ## dry run for `make clean` - print all untracked files
 clean: ## Clean the git work dir and remove all untracked files
 	# clean stage
 	git clean -xdf -- test/hack/bin test/hack/output test/hack/checkLabels
-	git checkout -- helm/prometheus-rules/Chart.yaml
-	git checkout -- helm/prometheus-rules/values.yaml
```

Comment:
Did you put the cleanup back somewhere else?

Reply:
The app was moved to app-build-suite so this is not needed

Reply:
Oh, so this is not directly related to this PR! Fine with me!

Reply:
Nope, sorry, just found it while working on this